doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -358,7 +358,7 @@ Parametric polymorphism
 Unlike ML, polymorphism in core Gallina is explicit: the type
 parameters of polymorphic functions must be declared explicitly, and
 supplied at each point of use. However, Coq provides two features to
-suppress redundant parameters:
+suppress redundant parameters.
```

Comment:
As mentioned elsewhere, I think the colon is preferable.

Reply:
My understanding is that a colon should be followed by a sequence of nominal groups, separated by semicolons, not sentences. The bullets have no grammatical meaning; they are only used to improve readability.

Reply:
There's a bit more to bullet points, see https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/, which recommends a period for this case.